PDF2IMG/index.html

```diff
@@ -19,6 +19,8 @@
   <h1>Datalogics PDF2IMG Documentation</h1>
  <h2>PDF2IMG</h2>
  <p>A command-line utility and API that converts PDF files to a variety of image formats, including PNG, JPG, TIFF, BMP, and more. The product is built upon the Adobe PDF Library and uses Adobe technology for unrivaled color management during the PDF conversion process. While PDF2IMG can be run with as few as two command line arguments, many other optional command line arguments are provided so that you can create custom output. PDF2IMG is available on both Windows and Linux platforms, 32 and 64 bit.</p>
+ <br>
+ <p>PDF2IMG's C# interface is now available as a x64 Windows NuGet package.</p>
```

Comment:
should this be "as an x64"?

Reply:
No, I think "as a x64" is correct because the word following  "a" isn't a vowel.

Reply:
but the syllable is a vowel sound - "Ex"